packages/docs-app/src/styles/_sections.scss

```diff
@@ -4,13 +4,13 @@
 @import "~@blueprintjs/core/src/common/variables";
 
 // Generate a selector for a page ID by reference
-@function page($ref, $comparator: "=") {
-  @return "[data-page-id#{$comparator}\"#{$ref}\"]";
+@function page($ref, $comparator: "$=") {
```

Comment:
`$=` because the most specific page is on the right side. prior to this change, the [selector on line 18](https://github.com/palantir/blueprint/pull/2489/commits/8303c58ae777d984cd7fc5b95d95032828671286#diff-b327cc76f467827a58306dcf634d2e0dR18) didn't work